pulp_rpm/app/models/modulemd.py

```diff
@@ -63,14 +66,16 @@ class Modulemd(Content):
     packages = models.ManyToManyField(Package)
     profiles = models.JSONField(default=dict)
     description = models.TextField()
+    digest = models.TextField()
 
     snippet = models.TextField()
+    repo_key_fields = ("name", "stream", "version", "context", "arch")
```

Comment:
@ggainey I think we want to include `static_context` too yes?  Or does it not matter?

I think in practice it wouldn't matter since the context differs but in theory it could.  And that's kind of how we got here, so maybe include it?

Reply:
static_context can be unset based on specs, so I would not include it